skimage/morphology/_flood_fill.py

```diff
@@ -98,30 +99,20 @@ def flood_fill(image, seed_point, new_value, *, selem=None, connectivity=None,
            [5, 5, 5, 5, 2, 2, 5],
            [5, 5, 5, 5, 5, 5, 3]])
     """
-    # Correct start point in ravelled image - only copy if non-contiguous
-    image = np.asarray(image)
-    copied = False
+    if inplace and not image.flags.contiguous:
+        # Flood fill requires some sort of contiguity - this makes a copy
+        _image = np.ascontiguousarray(image)
+    else:
+        _image = image
```

Comment:
Is this actually needed at all? `flood` makes its own contiguity check and copy if necessary.

Reply:
No, I think you're right. Nice!